src/main/java/com/plaid/client/request/common/Product.java

```diff
@@ -12,4 +12,5 @@
   @SerializedName("transactions") TRANSACTIONS,
   @SerializedName("investments") INVESTMENTS,
   @SerializedName("liabilities") LIABILITIES,
+  @SerializedName("payment") PAYMENT,
```

Comment:
Should this be `payment_initiation`? Same question about the payment model route names

Reply:
⚡️opening a new PR to address this